gapic/src/main/com/google/gapid/widgets/TabArea.java

```diff
@@ -47,7 +48,8 @@ public void onTabCreated(TabInfo tab) {
 
       @Override
       public void onTabClosed(TabInfo tab) {
-        analytics.postInteraction(tab.view, ClientAction.Disable);
+        analytics.postInteraction(tab.view,
```

Comment:
This is leaking an implementation detail out of the `TabComposite`. Just pass a boolean `pinned` to the `onTabClosed` event.

Reply:
Yes, thanks. I have added the function TabInfo.isPinned() to hide the implementation detail.